src/Components/DrillList.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { StyleSheet, View, Text, Image, TouchableOpacity, FlatList } from 'react-native';
+
+import * as list from '../styles/list.style';
+
+const DrillList = props => {
+  const drillsToDisplay = props.drillsToDisplay;
+  const navigation = props.navigation;
+  console.log(navigation);
+
+  return (
+    <View>
+      <FlatList
+        data={drillsToDisplay}
+        keyExtractor={item => item.id.toString()}
+        renderItem={({ item }) => (
+          <TouchableOpacity style={styles.drill} onPress={() => navigation.navigate('DrillPage', { drill: item })}>
```

Comment:
You should use destructuration for `styles` and `item`

Reply:
And `renderItem`, it might be interesting to write a proper function for this one

Reply:
done